src/components/App.js

```diff
@@ -1,25 +1,15 @@
-import React, { Component }from 'react';
```

Comment:
There's nothing wrong with this component. I just changed it to a functional component just to use less code